dynamics-nav/SETFILTER-Function--FieldRef-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETFILTER Function (FieldRef)"
+description: SETFILTER Function (FieldRef) Assigns a filter to a field that you specify. This topic contains information on the SETFILTER Function (FieldRef) syntax, parameters, remarks, and examples.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.